code/src/components/MainDish.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const MainDish = ({label, mainDish, onMainDishChange}) => {
+   
+    return (
+        <div className="main-dish">
+            <label htmlFor="mainDish">
+              <h1 tabindex="0" className='maindish-text'>What is your favorite main dish?</h1>
+              </label>
+            <select 
+                className='maindish-menu'
+                name={label}
+                value={mainDish}
+                onChange={onMainDishChange}
+                >
+                 <option>pasta</option>
+                 <option>meat</option>
+                 <option>seitan</option>
+                 <option>chicken</option>
```

Comment:
When actively choosing a dish the users choices are nicely printed in the summary 😃, but when the user doesn't make an active decision and leave the default choice (pasta) the state doesn't update and "pasta" is not printed in the summary. 
I checked my code and I have added something like: 
<option value=' '>Welcome to choose your dish</option> above my "pasta" option. I'm not sure if my solution is best practice though. However, Van provides another solution, using "disabled" for the first <option>, with no text. He describes it in Week10, lecture#2 about 19 minutes in 😃.